plugins/uploadimage/plugin.js

```diff
@@ -147,4 +153,15 @@
 	 * @cfg {String} [imageUploadUrl='' (empty string = disabled)]
 	 * @member CKEDITOR.config
 	 */
+
+	/**
+	 * This setting allows to configure custom types for image upload. If not provided default types will be used.
+	 * Warning: some browsers does not support all custom types eg. Google Chrome and Firefox cannot display `TIF` images.
+	 *
+	 * @since 4.8.1
+	 * @cfg {String[]} [uploadImage_supportedTypes=['jpeg', 'png', 'gif', 'bmp'] (empty array = disabled)]
+	 * @member CKEDITOR.config
+	 */
+	CKEDITOR.config.uploadImage_supportedTypes = [ 'jpeg', 'png', 'gif', 'bmp' ];
```

Comment:
Note that the name of the plugin should be lowercase (so `uploadimage`, not ~`uploadImage`~).

Reply:
@mlewand what do you think about variable name?